test_utils/utils.go

```diff
@@ -187,6 +208,41 @@ func CreateContainer(netconf string, k8sName string, ip string) (container_id, n
 	return
 }
 
+// Executes the Calico CNI plugin and return the error code of the command.
+func CmdDel(netconf, netnspath, name string) (exitCode int, err error) {
```

Comment:
Do both CmdDel and DeleteContainer need to exist? There's a lot of common code between them, should one call the other? Should other tests be updated to use CmdDel?